### ISSUE TITLE
Add CMake/Autotools build options for define XML_ATTR_INFO

### ISSUE THREAD

**Issue by hartwork** [libexpat/libexpat#244]
While writing https://github.com/libexpat/libexpat/pull/243/files#r246944739 I realized that the CI does not cover off-by-default build option / define `XML_ATTR_INFO`. So we may break that code without noticing or may have broken that code already. Let's cover it by the CI.

**Comment by MohammedKhajapasha** [comment]
hartwork, is this issue fixed ?

**Comment by hartwork** [comment]
It's unfixed:
```
# git --no-pager grep -l XML_ATTR_INFO
Changes
doc/reference.html
lib/expat.h
lib/xmlparse.c
```

**Comment by MohammedKhajapasha** [comment]
harttwork please review the change #264 updated with review comments

**Comment by hartwork** [comment]
Limiting this ticket to new build options new, CI aspect is now #273.

**Comment by hartwork** [comment]
Fixed by #264, closing.
